### ISSUE TITLE
CVE-2024-55565 vulnerability

### ISSUE THREAD

**Issue by SystemInfected** [webpack-contrib/css-loader#1623]
Hello! I didn't see any issues reporting this.

There is a vulnerability in nanoid@3.3.7 so this package should be upgraded to mitigate this.

Reference:
https://access.redhat.com/security/cve/cve-2024-55565

**Comment by alexander-akait** [comment]
We can't fix it here, sorry, we don't use `nanoid` here

**Comment by SystemInfected** [comment]
Yes that's true, but it is a dependency of a lot of dependencies used by `css-loader` that should probably be updated

![image](https://github.com/user-attachments/assets/cbba45d0-5428-4057-929e-2ff3e384d4e3)

**Comment by alexander-akait** [comment]
You can update these dependencies locally, we use `^` everywhere https://github.com/webpack-contrib/css-loader/blob/master/package.json#L57 and we can't update dependencies in transitive dependencies
